tbx/core/blocks.py

```diff
@@ -171,7 +171,7 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/video_block.html"
 
 
-class CallToActionStructValue(blocks.StructValue):
+class ButtonLinkStructValue(blocks.StructValue):
```

Comment:
This class is used by the `CallToActionBlock`, which is part of the core `StoryBlock` so I'm a bit surprised this isn't generating more migrations. I guess maybe updating a `value_class` option doesn't create a migration?

Reply:
> I guess maybe updating a value_class option doesn't create a migration

Precisely :). I felt this name would be more appropriate given what the class is doing

Reply:
> The reason I ask is because having ButtonLinkStructValue as the value class of two completely different things, which only share a 'field' feels odd, and it feels more natural for that behaviour to belong with the field/attribute/block directly.

You are right, which is why I had originally wanted to create subclasses for both block values inheriting from `ButtonLinkStructValue`, doing it the proper OOP way. 

I had also thought of doing it the way you suggested, but that meant changing a few other templates, which felt like too much out of the scope for this ticket.

> I don't know whether StructValues attached to sub blocks are available in the template, so this may not be possible.

They are, check [definition](https://github.com/torchbox/torchbox.com/blob/74d12aa737443817fa6c372fa8fcbd5063f0b396/tbx/core/blocks.py#L346) and [usage](https://github.com/torchbox/torchbox.com/blob/74d12aa737443817fa6c372fa8fcbd5063f0b396/tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/promo_block.html#L8-L13)

Reply:
> They are, check [definition](https://github.com/torchbox/torchbox.com/blob/74d12aa737443817fa6c372fa8fcbd5063f0b396/tbx/core/blocks.py#L346) and [usage](https://github.com/torchbox/torchbox.com/blob/74d12aa737443817fa6c372fa8fcbd5063f0b396/tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/promo_block.html#L8-L13)
Ah, I guess you're right. I was thinking of a custom defined `value_class`, but of course this is going to apply to the default StructValue too.